cocos/asset/asset-manager/downloader.ts

```diff
@@ -262,6 +262,18 @@ export class Downloader {
      */
     public downloadScript = downloadScript;
 
+    /**
+     * @engineInternal
+     * @internal
```

Comment:
Why need `@internal`?

Reply:
![image](https://github.com/cocos/cocos-engine/assets/28807867/3c145c83-a2aa-4edc-b43d-700fd3d12844)

Reply:
But only `engineInternal` does the thing. `internal` can not work.
@PPpro could you please double check it?

Reply:
For now, we need @internal to control docs generator.

Reply:
we've fixed the issue here https://github.com/cocos/cocos-engine/pull/15826

now we can use engineInternal to control both `.d.ts` and api doc generation @holycanvas